test/integration/integration_test.go

```diff
@@ -45,239 +75,230 @@ func newDescriptorStatusLegacy(
 	}
 }
 
-// TODO: Once adding the ability of stopping the server in the runner (https://github.com/envoyproxy/ratelimit/issues/119),
-//  stop the server at the end of each test, thus we can reuse the grpc port among these integration tests.
+func makeSimpleRedisSettings(redisPort int, perSecondPort int, perSecond bool, localCacheSize int) settings.Settings {
+	s := defaultSettings()
+
+	s.RedisPerSecond = perSecond
+	s.LocalCacheSizeInBytes = localCacheSize
+	s.BackendType = "redis"
+	s.RedisUrl = "localhost:" + strconv.Itoa(redisPort)
+	s.RedisPerSecondUrl = "localhost:" + strconv.Itoa(perSecondPort)
+	return s
+}
+
 func TestBasicConfig(t *testing.T) {
 	common.WithMultiRedis(t, []common.RedisConfig{
-		{Port: 6379},
```

Comment:
I also changed redis port 6379 to 6383 since 6379 is a traditional default port to run redis on and people might already have an instance running.